x-pack/plugins/session_view/public/components/detail_panel_alert_list_item/index.tsx

```diff
@@ -98,7 +111,19 @@ export const DetailPanelAlertListItem = ({
       buttonContent={
         <EuiText css={styles.alertTitleContainer} size="s">
           <p css={styles.alertTitle}>
-            <EuiIcon color="danger" type="alert" css={styles.alertIcon} />
+            <EuiToolTip
+              position="top"
+              content={`${capitalize(processEventAlertCategory)} ${
+                capitalize(processEventAlertCategory) !== 'alert' ? 'alert' : ''
```

Comment:
maybe we can make 'alert' a const?

Reply:
Yes, that's a good idea  since `alert` used multiple places. I create small util function too.